browser-interface/packages/shared/profiles/sagas/content/requestProfile.ts

```diff
@@ -1,11 +1,10 @@
 import defaultLogger from 'lib/logger'
 import type { RemoteProfile } from 'shared/profiles/types'
-import { ensureRealmAdapter } from 'shared/realm/ensureRealmAdapter'
 
 export async function requestProfile(userId: string, version?: number): Promise<RemoteProfile | null> {
-  const backendConfiguration = await ensureRealmAdapter()
+  const lambdasServer = 'https://peer.decentraland.org/lambdas'
```

Comment:
the `realmAdapter.services.lambdasServer` has the lambdas url for the connected realm, maybe it can be used here? not quite sure

Reply:
I see! I'm not an expert on this project at all, but it seems to me if you change this here it will use this all the time, please correct me if I'm wrong. If that's the case maybe we should leave the load balancer for the onboarding flow but the realm lambdas URL for everything else. Keep in mind the load balancer only works with foundation nodes, so it won't hit use the other nodes.